### DISCUSSION TITLE
How to Hide specific videos

### DISCUSSION THREAD

**Discussion by lamborghinipth** [FreeTubeApp/FreeTube#5426] [Q&A]
I know FreeTube has a filter rule feature to hide videos based on title. However, I have a case to hide some specific videos (not hide all videos of channel, just one or some videos). I do not know how to do it.

Please help me!

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Unfortunately that is not possible at the moment and this issue is being tracked in https://github.com/FreeTubeApp/FreeTube/issues/1362

**Reply by lamborghinipth** [reply]
I saw. When will they develop this feature?
